XIVSlothCombo/Combos/PvE/DRK.cs

```diff
@@ -59,6 +59,7 @@ public static class Buffs
             public const ushort
                 // Main Buffs
                 BloodWeapon = 742,
+                OlderDelirium = 1972,
```

Comment:
Just a suggestion,
Would Delirium 1972 and ScarletDelirium 3836 be better names?

Reply:
Scarlet Delirium would get me confused with the ability of the same name, I went for "Enhanced" in [b2b867a](https://github.com/Nik-Potokar/XIVSlothCombo/pull/1793/commits/b2b867a96a4159c0b762f82f949bb1be59b86ec9).